controllers/cloudflareCache.js

```diff
@@ -0,0 +1,92 @@
+const logsQuery = require("../models/logs");
+const cloudflare = require("../services/cloudflareService");
+const { logType } = require("../constants/logs");
+const { MAX_CACHE_PURGE_COUNT } = require("../constants/cloudflareCache");
+const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+
+/**
+ * Purges the Cache of Members Profile Page
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const purgeCacheByUserOrSuperUser = async (req, res) => {
+  try {
+    const { id, username, roles } = req.userData;
+    const logs = await logsQuery.fetchCacheLogs(id);
+    const logsCount = logs.length;
+
+    // Cache purged by a superuser without rate limits
+    if (roles.super_user) {
+      const { user } = req.body;
+      const files = [`https://members.realdevsquad.com/${user}`];
+      return purgeCache(res, id, files);
```

Comment:
where can I find this purge cache function?

Reply:
Just below this controller, there's another controller named as purgeCache